### ISSUE TITLE
Special queries like "\d+"

### ISSUE THREAD

**Issue by thousandsofthem** [jakob/Postico#43]
It would be really useful to be able to write and execute special queries like `\d+`
At this moment Postico yells on them

**Comment by jakob** [comment]
Yup, support for psql commands sure would be useful. At the moment all I can do is highlight them in red.

The problem is, these commands are implemented client side in psql. I'm not sure how I should add support for them without practically rewriting psql.

**Comment by thousandsofthem** [comment]
Thanks. Kinda sad :(
Well, how about menu with "special queries"?
Like "list tables+indexes(with owner info, sizes etc"
And pick these commands by hand
(Source code is available as reference, e.g.https://github.com/postgres/postgres/blob/c3cc844feb4a52fc57f53d129f33782d55f29340/src/bin/psql/describe.c#L676)

**Comment by bountin** [comment]
+1!

**Comment by thousandsofthem** [comment]
okay, i think it's better idea:

add general-purpose "bookmarks" (store label + query), similar to history thing
